isatools/net/mtbls2assaydict.py

```diff
@@ -0,0 +1,423 @@
+from __future__ import absolute_import
+import os
+import csv
+from collections import OrderedDict
+import pandas as pd
+import json
+import rdflib
+
+from rdflib import *
```

Comment:
This is a duplicate of the line above, hence not needed. I would also avoid asterisk imports from now on as they are discouraged
```suggestion
from rdflib import Graph # etc.
```